Source/WebCore/PAL/pal/spi/cocoa/WebPrivacySPI.h

```diff
@@ -163,6 +163,17 @@ typedef void (^WPRestrictedOpenerDomainsCompletionHandler)(NSArray<WPRestrictedO
 @end
 #endif
 
+#if !defined(HAS_WEB_PRIVACY_RESOURCE_MONITOR_URLS_API)
+@class WKContentRuleList;
+@class WKContentRuleListStore;
+
+typedef void (^WPRuleListPreparationCompletionHandler)(WKContentRuleList *, bool, NSError *);
+
+@interface WPResources (Staging_137691088)
+- (void)prepareResouceMonitorRulesForStore:(WKContentRuleListStore *)store completionHandler:(WPRuleListPreparationCompletionHandler)completionHandler;
+@end
+#endif
```

Comment:
All of the types and declarations here should exactly match what's declared in WebPrivacy. The intention of these types of "staging" things is so that you can continue building WebKit for some time on an old SDK.

The WebPrivacy in the new SDK will define `HAS_WEB_PRIVACY_RESOURCE_MONITOR_URLS_API`. So when you use the new SDK, all of this staging stuff gets ignored. (Actually now that I look at your WebPrivacy patch, it seems like you're missing that, so I'll open up a PR to add that declaration). Also, by convention, in `Staging_XYZ` the XYZ should match the radar number for the change being staged.

This should look something like:

```
@interface WPResources (Staging_141644757)
- (void)loadResouceMonitorRulesForStore:(WKContentRuleListStore *)store completionHandler:(WPRuleListPreparationCompletionHandler)completionHandler;
@end
```